src/Redaxtor.js

```diff
@@ -282,6 +289,22 @@ class Redaxtor {
             this.barNode
         );
         options.navBarRoot.appendChild(this.barNode);
+
+    }
+    /**
+     * Renders tbe hover overlay
+     */
+    showHoverOverlay(root){
+        this.overlayNode = document.createElement("DIV");
```

Comment:
Or. I'll replace  it but it is copy of showBar method where is div used too. It will be necessary replace this in future